os/include/debug.h

```diff
@@ -153,8 +153,14 @@ Once LOGM is approved, each module should have its own index
  * @details @b #include <debug.h>
  * @since TizenRT v1.0
  */
```

Comment:
Can you please make separate commit for this temporary change, we can revert after the issue fix

Reply:
Done.